pkgs/development/libraries/vte/default.nix

```diff
@@ -61,6 +63,7 @@ stdenv.mkDerivation rec {
 
   buildInputs = [
     fribidi
+    gobject-introspection
```

Comment:
I am still unconvinced gobject-introspection should ever go to `buildInputs` for libraries. I think here, we should rather remove this line https://gitlab.gnome.org/GNOME/vte/-/blob/fb604fe287abf64645a83ef27e2c05da14fbbae0/bindings/gir/meson.build#L17 since meson will already check for GIR.

Reply:
Even when removing this line it tries to find gir somewhere else:

``` 
vte-aarch64-unknown-linux-gnu> Found CMake: NO
vte-aarch64-unknown-linux-gnu> Run-time dependency gobject-introspection-1.0 found: NO (tried pkgconfig)
vte-aarch64-unknown-linux-gnu>
vte-aarch64-unknown-linux-gnu> bindings/gir/meson.build:27:2: ERROR: Dependency "gobject-introspection-1.0" not found, tried pkgconfig
vte-aarch64-unknown-linux-gnu>
vte-aarch64-unknown-linux-gnu> A full log can be found at /build/vte-0.68.0/build/meson-logs/meson-log.txt
```

Reply:
Yeah, I think that is a Meson issue, see https://github.com/NixOS/nixpkgs/pull/177538#discussion_r896125213.

Reply:
What do you suggest to fix this? Does this need to be fixed by meson?

Reply:
Yeah, unless I am mistaken, I think it should be fixed by Meson.

Reply:
Will be unnecessary after https://github.com/NixOS/nixpkgs/pull/189215 is in master